From de2d4909806f2d8966a4170b7f62de66df53efb3 Mon Sep 17 00:00:00 2001
From: Meng Li <Meng.Li@windriver.com>
Date: Tue, 30 Mar 2021 16:19:45 +0800
Subject: [PATCH] driver: drm: vc4: fix build issue

When merge code from upstream kernel tree, some building
issues are involved. So, create patch to fix it.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/vc4/vc4_hdmi.c      | 4 ++--
 drivers/gpu/drm/vc4/vc4_hdmi_regs.h | 2 --
 2 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/drivers/gpu/drm/vc4/vc4_hdmi.c b/drivers/gpu/drm/vc4/vc4_hdmi.c
index ea1e8de0c4b9..3b61914fd46c 100644
--- a/drivers/gpu/drm/vc4/vc4_hdmi.c
+++ b/drivers/gpu/drm/vc4/vc4_hdmi.c
@@ -2043,7 +2043,7 @@ static void vc4_cec_read_msg(struct vc4_hdmi *vc4_hdmi, u32 cntrl1)
 					VC4_HDMI_CEC_REC_WRD_CNT_SHIFT);
 
 	if (msg->len > 16) {
-		DRM_ERROR("Attempting to read too much data (%d)\n", msg->len);
+		drm_err(dev, "Attempting to read too much data (%d)\n", msg->len);
 		return;
 	}
 	for (i = 0; i < msg->len; i += 4) {
@@ -2145,7 +2145,7 @@ static int vc4_hdmi_cec_adap_transmit(struct cec_adapter *adap, u8 attempts,
 	unsigned int i;
 
 	if (msg->len > 16) {
-		DRM_ERROR("Attempting to transmit too much data (%d)\n", msg->len);
+		drm_err(dev, "Attempting to transmit too much data (%d)\n", msg->len);
 		return -ENOMEM;
 	}
 	for (i = 0; i < msg->len; i += 4)
diff --git a/drivers/gpu/drm/vc4/vc4_hdmi_regs.h b/drivers/gpu/drm/vc4/vc4_hdmi_regs.h
index 36c37aed4332..149b4b4cfdde 100644
--- a/drivers/gpu/drm/vc4/vc4_hdmi_regs.h
+++ b/drivers/gpu/drm/vc4/vc4_hdmi_regs.h
@@ -31,8 +31,6 @@ enum vc4_hdmi_field {
 	HDMI_CEC_CPU_MASK_STATUS,
 	HDMI_CEC_CPU_MASK_SET,
 	HDMI_CEC_CPU_MASK_CLEAR,
-	HDMI_CEC_CPU_STATUS,
-	HDMI_CEC_CPU_SET,
 
 	/*
 	 * Transmit data, first byte is low byte of the 32-bit reg.
-- 
2.17.1

